portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +10,16 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
-
+  
+  String fact[] = new String[2];
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    fact[0] = " { \"hobby\" : \"I love basketball\" }";
+    fact[1] = " { \"quote\" : \"My favorite quote is Being the richest man in the cemetery doesn’t matter to me. Going to bed at night saying we’ve done something wonderful… that’s what matters to me.\" }";
+    // Convert the server stats to JSON
```

Comment:
Remove this comment - it doesnt belong anymore since you aren't using server stats here.

Reply:
Done.